public/app/features/explore/RichHistory/RichHistory.test.tsx

```diff
@@ -21,30 +20,23 @@ const setup = (propOverrides?: Partial<RichHistoryProps>) => {
 
   Object.assign(props, propOverrides);
 
-  const wrapper = mount(<RichHistory {...props} />);
-  return wrapper;
+  render(<RichHistory {...props} />);
 };
 
 describe('RichHistory', () => {
-  it('should render all tabs in tab bar', () => {
-    const wrapper = setup();
-    expect(wrapper.find(Tab)).toHaveLength(3);
+  it('should tabs as defined', async () => {
+    setup();
+    const tabs = await screen.getAllByLabelText(/Tab*/);
```

Comment:
```suggestion
  it('should tabs as defined', () => {
    setup();
    const tabs = screen.getAllByLabelText(/Tab*/);
```
`getBy*` queries are sync, so no need for `await` here.